src/docs/files-provider-deprecation.rst

```diff
@@ -0,0 +1,60 @@
+Files Provider Removal
+######################
+
+The SSSD team has announced the `removal of the files provider <https://pagure.io/fesco/issue/3107>`_ feature in recent versions of SSSD in Fedora/CentOS Stream/RHEL. This document describes how to switch to using the SSSD 'proxy' provider to replace files provider functionality.
+
+Practically, there are only two use cases that currently justify usage of the files provider feature:
+
+* smart card authentication of local users
+* session recording for local users
+
+For both cases the proxy provider is a viable substitute. Otherwise, files provider functionality does not provide any benefit and should remain disabled.
```

Comment:
Maybe better something like "Otherwise there are no benefits in SSSD handling local users"?

Reply:
Fixed.